pkg/rhttp/datatx/manager/tus/tus.go

```diff
@@ -117,6 +124,7 @@ func (m *manager) Handler(fs storage.FS) (http.Handler, error) {
 					},
 					Path: utils.MakeRelativePath(filepath.Join(info.MetaData["dir"], info.MetaData["filename"])),
 				}
+				datatx.InvalidateCache(owner, ref, m.statCache)
```

Comment:
Putting the cache invalidation here means that it won't happen if event publishing is disabled. I don't think those should be tied to each other.

Reply:
correct, digging into it